Examples/Algorithms/Printers/ActsExamples/Printers/HitsPrinter.cpp

```diff
@@ -0,0 +1,132 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2019 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "HitsPrinter.hpp"
+
+#include "Acts/Plugins/Digitization/PlanarModuleCluster.hpp"
+#include "Acts/Utilities/Logger.hpp"
+#include "ActsExamples/EventData/GeometryContainers.hpp"
+#include "ActsExamples/EventData/IndexContainers.hpp"
+#include "ActsExamples/EventData/SimParticle.hpp"
+#include "ActsExamples/Framework/WhiteBoard.hpp"
+#include "ActsExamples/Utilities/Range.hpp"
+
+#include <vector>
+
+ActsExamples::HitsPrinter::HitsPrinter(
+    const ActsExamples::HitsPrinter::Config& cfg, Acts::Logging::Level level)
+    : BareAlgorithm("HitsPrinter", level), m_cfg(cfg) {
+  if (m_cfg.inputClusters.empty()) {
+    throw std::invalid_argument("Input clusters collection is not configured");
+  }
+  if (m_cfg.inputClusters.empty()) {
```

Comment:
The `m_cfg.inputClusters` should be `m_cfg.inputHitParticlesMap`?

Reply:
Yes, it should. Thanks for catching that. Fixed.